portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase

```diff
@@ -36,6 +36,53 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPP-8122. Verify that when remote publishing, the child pages are not published when they are unselected."
+	@priority = "3"
+	test AssertCanotPublishChildPageWhenUnselected {
+		property test.name.skip.portal.instance = "RemoteStaging#AssertCanotPublishChildPageWhenUnselected";
+
+		task ("Given: two children pages for 'Staging Test Page'") {
+			for (var i : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Site Name",
+					layoutName = "Test Child Page ${i}",
+					parentLayoutName = "Staging Test Page");
+			}
+		}
+
+		task ("When: In Remote Publish settings, select only the parent Page and not the child pages and publish to remote live.") {
```

Comment:
Summarize at the business level, no need to write down every action.
e.g. 
"When: Publish only the Parent Page and not the child pages to remote live"